utilities/cli-runner/src/main/kotlin/org/jetbrains/kotlin/cli/utilities/main.kt

```diff
@@ -29,6 +29,10 @@ private fun mainImpl(args: Array<String>, konancMain: (Array<String>) -> Unit) {
             defFileDependencies(utilityArgs)
         "generatePlatformLibraries" ->
             generatePlatformLibraries(utilityArgs)
+
+        "llvm" -> runLlvmTool(utilityArgs)
+        "clang" -> runLlvmClangToolWithTarget(utilityArgs)
```

Comment:
It might break user's scripts. I suggest to add a prefix (e.g. `kotlin-`).

Reply:
No it can't because this PR doesn't add scripts to `dist/bin`.
It just adds subcommands to `run_konan`.